massa-execution-worker/Cargo.toml

```diff
@@ -91,7 +96,7 @@ massa_event_cache = { workspace = true }
 tempfile = { workspace = true, optional = true }
 massa_wallet = { workspace = true }
 massa-proto-rs = { workspace = true }
-schnellru = { workspace = true }
+schnellru = { workspace = true, optional = true }
```

Comment:
not necessary

Reply:
Removing "optional = true" here add a warning when compiling without any feature:

```
warning: extern crate `schnellru` is unused in crate `massa_execution_worker`
```

Reply:
can't we remove the whole schnellru ?

Reply:
schnellru is used with the feature execution-trace and need a lot of refactor to remove it. 

Used for LruMap enabled with execution-trace